src/components/molecules/ScheduleEvent/ScheduleEvent.scss

```diff
@@ -44,6 +45,38 @@ $description--margin-top: 2px;
     }
   }
 
+  &--short {
```

Comment:
(took a random line to comment)
As we discussed earlier the event info (description + host) should not appear in the center, but stick to the left.
\+ the width of `expandable` icon should not be different when the event box is expanded.  

this is how it's right now:
![recording new](https://user-images.githubusercontent.com/63313732/125538626-a24b371f-5f14-4781-9e7b-b953213ba163.gif)

the icon is floating depends on the description, but it should always appear at the same place relatively to the event box (just like the bookmark icon - it does not change its place)

Reply:
fixed 64c0c27